src/database.js

```diff
@@ -0,0 +1,166 @@
+const fs = require('fs');
+const {decode, encode} = require('./cipher');
+const path = require('path');
+
+// ID length in bytes
+const idSize = 2;
+
+const toBytes = (data) => {
+  const idBuffer = Buffer.alloc(idSize);
+  idBuffer.writeUInt16BE(data.lastIdx);
+  // Write json string with default UTF-8 encoding
+  const jsonBuffer = Buffer.from(JSON.stringify(data.entities));
```

Comment:
I'd rather divide this to enhance readability.
```suggestion
  const jsonStr = JSON.stringify(data.entities);
  
  const jsonBuffer = Buffer.from(jsonStr);
```

Reply:
Agree with you, though I don't really like the jsonStr name

Reply:
You can rename it as well. It was just the first thing that came up to my mind)